subiquity/server/controllers/drivers.py

```diff
@@ -74,6 +74,12 @@ def start(self):
     async def _list_drivers(self, context):
         with context.child("wait_apt"):
             await self._wait_apt.wait()
+        # At this point, the user can't go back all the way to the source
+        # screen to enable/disable the "search drivers" checkbox.
```

Comment:
Er. Why is that? Maybe extend the comment a bit. One day (maybe soon!) we'll allow the user to retry apt configuration if it fails, will that invalidate this comment?

Reply:
Well, `self._wait_apt.wait()` unblocks when we receive the `APT_CONFIGURED notification. 
Currently, we configure APT (and then broadcast the `APT_CONFIGURED` notification) after the user has "confirmed destructive changes" (i.e., finished partitioning). From the GUI standpoint, there is no going back beyond that point _yet?_. Also, by the time APT_CONFIGURED is received, the "source" is already mounted so we don't want the user to change it. I'll add some details to the comment.

Regarding the ability to retry the APT configuration, I'd still assume we wouldn't allow the user to change things in the sources screen, so it wouldn't invalidate the comment.

The `_list_drivers` task will probably need to be executed once more if we change the APT configuration though, but that's a different story.

Reply:
Thanks, I just wanted to make sure we're not backing ourselves into a corner!